newspaper/network.py

```diff
@@ -71,7 +71,9 @@ def get_html_2XX_only(url, config=None, response=None):
     return html
 
 
-def _get_html_from_response(response):
+def _get_html_from_response(response, config=None):
```

Comment:
final nit: can you change `config=None` to just config, making this parameter optional is dangerous because we don't check if `config` is None, for example in line 75, we directly access `config.ignored_content_types_defaults` which will be a null pointer if someone accidentally calls `get_html_from_response` without the config param

Since this function is only used twice in the codebase, in both instances you have added the config param in the callsite, I think it's fine to force config to be a required param